packages/figma-utils/README.md

```diff
@@ -10,39 +10,4 @@ For a list of supported commands and options, run:
 npx @sit-onyx/figma-utils@latest --help
 ```
 
```

Comment:
fyi I just noticed that this readme is the only one which has the package name as H1. do you want to align the readmes?

Reply:
They all have the package name as h1 🤔

Reply:
I mean the other ones have not @sit-onyx/figma-utils but like "Onyx Figma Utils" as headline.